test/e2e/tests/errors.spec.js

```diff
@@ -131,11 +383,10 @@ describe('Sentry errors', function () {
         },
         async ({ driver, mockedEndpoint }) => {
           await driver.navigate();
-          await driver.fill('#password', 'correct horse battery staple');
```

Comment:
The login step was producing indeterminacy in the state snapshots because we didn't wait for the login process to finish. Rather than adding a waiting step, the login was removed (it wasn't necessary for the test).